pkgs/os-specific/linux/lvm2/common.nix

```diff
@@ -35,44 +45,56 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [
-    libaio
-  ] ++ lib.optionals udevSupport [
-    udev
-  ] ++ lib.optionals (!onlyLib) [
-    libuuid
-  ] ++ lib.optionals enableVDO [
-    vdo
-  ];
+  buildInputs =
+    [
+      libaio
+    ]
+    ++ lib.optionals udevSupport [
+      udev
+    ]
+    ++ lib.optionals (!onlyLib) [
+      libuuid
+    ]
+    ++ lib.optionals enableVDO [
+      vdo
+    ];
 
-  configureFlags = [
-    "--disable-readline"
-    "--enable-pkgconfig"
-    "--with-default-locking-dir=/run/lock/lvm"
-    "--with-default-run-dir=/run/lvm"
-    "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
-    "--with-systemd-run=/run/current-system/systemd/bin/systemd-run"
-  ] ++ lib.optionals (!enableCmdlib && !onlyLib) [
-    "--bindir=${placeholder "bin"}/bin"
-    "--sbindir=${placeholder "bin"}/bin"
-    "--libdir=${placeholder "lib"}/lib"
-    "--with-libexecdir=${placeholder "lib"}/libexec"
-  ] ++ lib.optional enableCmdlib "--enable-cmdlib"
-  ++ lib.optionals enableDmeventd [
-    "--enable-dmeventd"
-    "--with-dmeventd-pidfile=/run/dmeventd/pid"
-    "--with-default-dm-run-dir=/run/dmeventd"
-  ] ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-    "ac_cv_func_malloc_0_nonnull=yes"
-    "ac_cv_func_realloc_0_nonnull=yes"
-  ] ++ lib.optionals udevSupport [
-    "--enable-udev_rules"
-    "--enable-udev_sync"
-  ] ++ lib.optionals enableVDO [
-    "--enable-vdo"
-  ] ++ lib.optionals stdenv.hostPlatform.isStatic [
-    "--enable-static_link"
-  ];
+  configureFlags =
+    [
+      "--disable-readline"
+      "--enable-pkgconfig"
+      "--with-default-locking-dir=/run/lock/lvm"
+      "--with-default-run-dir=/run/lvm"
+      "--with-systemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
+      "--with-systemd-run=/run/current-system/systemd/bin/systemd-run"
+      "--with-default-profile-subdir=profile.d"
```

Comment:
(this line is the added one)

Reply:
The latter, because otherwise CI complained, while the real change is hopefully pretty trivial.